pkgs/top-level/all-packages.nix

```
@@ -20399,6 +20399,8 @@ in
 
   hactool = callPackage ../tools/compression/hactool { };
 
+  hdhomerun_config_gui = callPackage ../applications/video/hdhomerun_config_gui { };  
```

I'd suggest
```suggestion
  hdhomerun-config-gui = callPackage ../applications/video/hdhomerun-config-gui { };  
```
even through the binary is using `_`. Using `-` is a bit more idiomatic in package names.